testbed/pyproject.toml

```diff
@@ -66,6 +66,7 @@ test_sources = [
 ]
 requires = [
     "../winforms",
+    "comtypes",
```

Comment:
While this fixes the immediate problem, it's not the right fix. You're proposing adding code to `toga-winforms` that uses `comtypes`; that means that `comtypes` is a dependency of `toga-winforms`, not the testbed. Testbed should be picking up comtypes transitively because of the dependency on `toga-winforms`.

Reply:
Oops, you're absolutely correct.